application/src/main/java/org/opentripplanner/street/search/state/State.java

```diff
@@ -92,7 +93,12 @@ public static Collection<State> getInitialStates(
     for (Vertex vertex : vertices) {
       for (StateData stateData : StateData.getInitialStateDatas(streetSearchRequest)) {
         states.add(
-          new State(vertex, streetSearchRequest.startTime(), stateData, streetSearchRequest)
+          new State(
+            vertex,
+            streetSearchRequest.startTime().truncatedTo(ChronoUnit.SECONDS),
```

Comment:
Maybe you should rather do the truncation in the street search request than here or would it cause problems in some other context?